File: C/2.c
#include <stdio.h>

int main(){
	int a = 1;
	int b = 2;
	int acc = 0;
	while(a < 4000000 || b < 4000000){
    		if(a < 4000000 && a%2 == 0){ 
			acc += a;
		}
    		if(b < 4000000 && b%2 == 0){
		       	acc += b;
		}
    		a += b;
    		b += a;
	}
	printf("%d\n", acc);
	return 0;
}

File: python/2.py
a = 1
b = 2
acc = 0
while a < 4000000 or b < 4000000:
    if a < 4000000 and not a%2: acc += a
    if b < 4000000 and not b%2: acc += b
    a += b
    b += a

print(acc)
